plugins/gpio/GPIODriver.cpp

```diff
@@ -134,7 +138,12 @@ bool GPIODriver::SetupGPIO() {
       break;
     }
 
-    GPIOPin pin = {pin_fd, UNDEFINED, false};
+    // Check if pin is in the inverted pin list
+    bool inverted = (find(m_options.gpio_inverted_pins.begin(),
+                          m_options.gpio_inverted_pins.end(),
+                          (*iter)) != m_options.gpio_inverted_pins.end());
```

Comment:
Can't you use our STLContains function?
http://docs.openlighting.org/ola/doc/latest/group__stl.html#ga6bc94dfb62cab831bfa2f0fc7e4daa04